src/Blackfire/Profile/Configuration.php

```diff
@@ -35,6 +40,11 @@ class Configuration
      */
     private $build;
 
+    public function __construct()
+    {
+        $this->assertManager = new AssertManager();
```

Comment:
What's about allowing people injecting their own instance?

```suggestion
    public function __construct($manager)
    {
        $this->assertManager = $manager ? $manager : new AssertManager();
```

Reply:
```php
public function __construct($manager = null)
```

And it could be type-hinted as well